stats/Exemplars.md

```diff
@@ -0,0 +1,48 @@
+Exemplars
+===
+
+Exemplars are example data points for aggregated data. In particular, for distribution-type
```

Comment:
Is this document proposes new API or event schema? Or it is simply a terminology description?

Reply:
There's already protobuf for this and an implementation in Java and I'm working on a Go implementation.

Reply:
do you know if this is part of OpenMetrics charter as well? If so - perhaps it should be less about traces and more about metrics

Reply:
+ @mtwo

Reply:
Yes it is part of OpenMetrics, they have support for exemplars in their definition.

Reply:
It's not only tracing-specific: this doc also proposes we add the tags as attachments.